osf/external/tasks.py

```diff
@@ -0,0 +1,19 @@
+from framework.celery_tasks import app as celery_app
```

Comment:
This file also needs to be added to `CeleryConfig.imports`, but I'll get that on merge